packit_service/service/events.py

```diff
@@ -719,11 +733,39 @@ def __init__(
         self.sender_login = sender_login
         self.status = status
 
+    @classmethod
+    def from_event_dict(cls, event: dict):
+        account_type = event.get("account_type")
+        account_login = event.get("account_login")
+        sender_login = event.get("sender_login")
+        account_id = event.get("account_id")
+        account_url = event.get("account_url")
+        sender_id = event.get("sender_id")
+        created_at = event.get("created_at")
+        installation_id = event.get("installation_id")
+        repositories = event.get("repositories")
+
+        return InstallationEvent(
+            installation_id=installation_id,
+            account_login=account_login,
+            account_id=account_id,
+            account_url=account_url,
+            account_type=account_type,
+            created_at=created_at,
+            repositories=repositories,
+            sender_id=sender_id,
+            sender_login=sender_login,
+        )
```

Comment:
I am thinking if we can avoid this manual forwarding.

At least, what about getting rid of the one-time variables and fill the attributes directly? Using `get` should be safe:

```suggestion
        return InstallationEvent(
            installation_id=event.get("installation_id"),
            account_login=event.get("account_login"),
            account_id=event.get("account_id"),
            account_url=event.get("account_url"),
            account_type=event.get("account_type"),
            created_at=event.get("created_at"),
            repositories=event.get("repositories"),
            sender_id=event.get("sender_id"),
            sender_login=event.get("sender_login"),
        )
```

Or what about expanding the dictionary? Are there any unused values?


```suggestion
        return InstallationEvent(**event)
```

Reply:
using `InstallationEvent(**event)` would need to pop more attributes from the event dict before, so I changed it at least without using the one-time variables